src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionResolveTests.cs

```diff
@@ -108,7 +108,7 @@ void M()
             {
                 GenerateTextEdit(@"private const int V = 1;
 
-", new LSP.Range { Start = new Position(2, 4), End = new Position(2, 4) }),
+    ", new LSP.Range { Start = new Position(2, 4), End = new Position(2, 4) }),
```

Comment:
I switched the VerifyJson helpers to not remove whitespace, instead use the shared AssertToleratingWhitespaceDifferences helper. That caught this issue, where the actual returned edit has 4 spaces here for the method indentation.